core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -345,8 +350,8 @@ public void close() {
         sqlExecutionContext.with(AllowAllCairoSecurityContext.INSTANCE, null, null, -1, null);
         Misc.free(path);
         Misc.free(utf8Sink);
-        Misc.free(circuitBreaker);
         freeBuffers();
+        registry.release(circuitBreakerId);
```

Comment:
Please comment on why we don't close `circuitBreaker` instance here so that it won't be added back in the future.

Reply:
Done in fd45433e .